requirements.txt

```diff
@@ -13,3 +13,5 @@ fusepy==2.0.4
 python-dateutil==2.8.1
 diffimg==0.2.3
 selenium==3.141.0
+
+requests~=2.24.0
```

Comment:
Can you explain what this library is for?

Reply:
There are a few files importing this library, like codalab/rest/interpret.py, codalab/worker/docker_utils.py, etc